service/alerter/aws.go

```diff
@@ -1,10 +1,32 @@
 package alerter
 
 import (
+	"fmt"
+
 	awsresources "github.com/giantswarm/aws-operator/resources/aws"
 	"github.com/giantswarm/microerror"
 )
 
+// ListMasterInstances lists the master instances for this cluster.
```

Comment:
This uses the legacy `resources/aws` package. But the bug only affects legacy so we can delete this check when we retire the code.